apps/web/src/app/admin-console/organizations/reporting/organization-reporting-routing.module.ts

```diff
@@ -81,6 +88,20 @@ const routes: Routes = [
           },
         ],
       },
+      // Event routing is temporarily duplicated
+      {
+        path: "events",
+        component: EventsComponent,
+        canMatch: [breadcrumbEventLogsPermission$], // if this matches, the flag is ON
+        canActivate: [
+          organizationPermissionsGuard(
+            (org) => org.canAccessEventLogs || org.useEvents || org.isOwner,
```

Comment:
This should match the logic in the layout component:

```suggestion
            (org) => (org.canAccessEventLogs && org.useEvents) || org.isOwner,
```

As it's written right now, any org member can access if the org's plan includes events, which is not what we want.